api/src/pcapi/core/bookings/models.py

```diff
@@ -74,6 +74,12 @@ class BookingExportType(enum.Enum):
     EXCEL = "excel"
 
 
+class BookingAuthorType(enum.Enum):
+    OFFERER = "partenaire culturelle"
+    BACKOFFICE = "backoffice"
+    AUTO = "automatique (event)"
```

Comment:
```suggestion
    AUTO = "automatique"
```
Peut-être qu'un jour, autre chose que les events seront validés automatiquement.